apps/finance/app/src/components/Transfers.js

```diff
@@ -89,19 +89,14 @@ const getDownloadData = async (transfers, tokenDetails, resolveAddress) => {
       }
     )
   )
-  const csvContent = [
-    'data:text/csv;charset=utf-8,Date,Name,Source/Recipient,Reference,Amount',
-  ]
-    .concat(mappedData)
-    .join('\n')
-  return window.encodeURI(csvContent)
+  return ['Date,Name,Source/Recipient,Reference,Amount'].concat(mappedData)
```

Comment:
I think the `.join()` being here is a bit more clear

Reply:
That's where it was originally but it made more sense to me to keep the function as a function that returns an array instead of a string, what do you think? should I change it?

Reply:
Depends what data format you think "data" should be; I felt like a string made more sense here (as the export "data") but if an array feels better to you let's leave it :).